configs/milvus.yaml

```diff
@@ -307,6 +308,13 @@ dataCoord:
     missingTolerance: 86400 # file meta missing tolerance duration in seconds, 60*24
     dropTolerance: 86400 # file belongs to dropped entity tolerance duration in seconds, 60*24
 
+  bindIndexNodeMode:
```

Comment:
delete bindIndexNodeMode from indexCoord

Reply:
will delete it next pr.